core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -106,22 +109,28 @@ export class ContributionsAndReview
   SUGGESTION_TYPE_TRANSLATE: string;
   TAB_TYPE_CONTRIBUTIONS: string;
   TAB_TYPE_REVIEWS: string;
+  TAB_TYPE_ACCOMPLISHMENTS: string;
   activeExplorationId: string;
   contributions: Record<string, SuggestionDetails> | object;
   userDetailsLoading: boolean;
   userIsLoggedIn: boolean;
   activeTabType: string;
   activeSuggestionType: string;
+  activeAccomplishmentType: string;
   dropdownShown: boolean;
+  isAccomplishmentsTab: boolean;
   activeDropdownTabChoice: string;
   reviewTabs: TabDetails[] = [];
+  accomplishmentsTabs: TabDetails[] = [];
   contributionTabs: TabDetails[] = [];
   tabNameToOpportunityFetchFunction: {
     [key: string]: {
       [key: string]: Function;
     };
   };
 
+  accomplismentsEnabled: boolean = false;
```

Comment:
accomplishments is misspelled, and I also don't know what this variable name means (what does it means for "accomplishments to be enabled"). How does it differ from "isAccomplishmentsTab"?